README.rst

```diff
@@ -153,20 +153,23 @@ Developers". Feel free to cite our work in your research using the following Bib
 
 .. code-block::
 
-    @software{uieda_leonardo_2020_4025418,
+    @software{pygmt_2021_4522136,
       author       = {Uieda, Leonardo and
                       Tian, Dongdong and
                       Leong, Wei Ji and
                       Toney, Liam and
+                      Schlitzer, William and
+                      Grund, Michael and
                       Newton, Tyler and
+                      Jones, Meghan and
```

Comment:
```suggestion
                      Ziebarth, Malte and
                      Jones, Meghan and
```

Need to add Malte too.

And just checking with @meghanrjones can I put down you and Paul's affiliation as "Department of Earth Sciences, School of Ocean and Earth Science and Technology, University of Hawai‘i at Mānoa" in Zenodo?

Reply:
Yes, sounds good.